scripts/in_container/bin/install_aws.sh

```diff
@@ -35,7 +35,11 @@ if command -v aws; then
     exit 1
 fi
 
-DOWNLOAD_URL="https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip"
+if [[ $(uname -m) == "arm64" || $(uname -m) == "aarch64" ]]; then
```

Comment:
```suggestion
HOST=$(uname -m)
if [[ ${HOST} == "arm64" || ${HOST} == "aarch64" ]]; then
```

Reply:
@feluelle I have applied this change.Could you please review?